tests/cross/src/test/scala/tests/hover/HoverLiteralSuite.scala

```diff
@@ -0,0 +1,74 @@
+package tests.hover
+
+import tests.pc.BaseHoverSuite
+
+class HoverLiteralSuite extends BaseHoverSuite {
+
+  check(
+    "literal-int",
```

Comment:
@olafurpg Was there are reason for not adding hover over literals? This PR adds it but it can be reverted.

Reply:
Literals were intentionally excluded because they are verbose. The hover messages take up a lot of visual real estate on the screen and the hypothesis was that it’s mentally easy to infer the type of literals.

Reply:
As suggested, added it just for ranges.